engine-tests/src/main/java/org/terasology/testUtil/ModuleManagerFactory.java

```diff
@@ -35,6 +36,15 @@ private ModuleManagerFactory() {
 
     public static ModuleManager create() throws Exception {
         ModuleManager moduleManager = new ModuleManagerImpl("");
+        return LoadModuleEnvironment(moduleManager);
+    }
+
+    public static ModuleManager create(String masterServerAddress) throws Exception {
```

Comment:
Please add JavaDoc explaining what the `masterServerAddress` is used for.

Reply:
added javadoc